Contributing.MD

```diff
@@ -0,0 +1,30 @@
+Contributing to README Template
+
+We welcome your contributions to README Template. This doc describes the process to contribute patches to README Template and the general guidelines we expect contributors to follow.
+Before You Start
+
+We accept patches in the form of github pull requests. If you are new to github, please read How to create github pull requests first.
+Contributor License Agreements
+
+Contributions to this project must be accompanied by a Contributor License Agreement. You (or your employer) retain the copyright to your contribution, this simply gives us permission to use and redistribute your contributions as part of the project.
+If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an individual CLA.
+If you work for a company that wants to allow you to contribute your work, then you'll need to sign a corporate CLA.
+Contributing Process
+
+Most pull requests should go to the master branch and the change will be included in the next major/minor version release (e.g., 3.6.0 release). If you need to include a bug fix in a patch release (e.g., 3.5.2), make sure it’s already merged to master, and then create a pull request cherry-picking the commits from master branch to the release branch (e.g., branch 3.5.x).
+For each pull request, a Public Access Implementation and Support team member will be assigned to review the pull request. For minor cleanups, the pull request may be merged right away after an initial review. For larger changes, you will likely receive multiple rounds of comments and it may take some time to complete. We will try to keep our response time within 7-days but if you don’t get any response in a few days, feel free to comment on the threads to get our attention. We also expect you to respond to our comments within a reasonable amount of time. If we don’t hear from you for 2 weeks or longer, we may close the pull request. You can still send the pull request again once you have time to work on it.
+Once a pull request is merged, we will take care of the rest and get it into the final release.
+Pull Request Guidelines
+
+Create small PRs that are narrowly focused on addressing a single concern. We often receive PRs that are trying to fix several things at a time, but if only one fix is considered acceptable, nothing gets merged and both author's & review's time is wasted. Create more PRs to address different concerns and everyone will be happy.
+For speculative changes, consider opening an issue and discussing it first. 
```

Comment:
Think we should add an Issue section before the PR sections, describing when an Issue should be opened and any guidelines for submitting those.

I think Code.gov's [CONTRIBUTING file ](https://github.com/GSA/code-gov-front-end/blob/master/CONTRIBUTING.md)has a good one.